README.md

```diff
@@ -15,39 +15,72 @@
-2. Move the theme folder to **".themes"** in your home directory. **(~/.themes)** (Skip this step if you are using the AUR package)
-3. Select the downloaded theme via your desktop specific tweaks application (**gnome-tweaks** on Gnome 3+).
+### Installing the theme manually
 
-### For Arch Linux users
+1. Download and extract the theme zip from the [releases](https://github.com/catppuccin/gtk/releases/) page.
+2. Move the theme folder to the **~/.local/share/themes** directory (Skip this step if you're using the AUR package)
+3. Select the downloaded theme via your desktop specific tweaks application (GNOME Tweaks on GNOME 3+)
+4. To theme other apps that are using GTK, make sure to run the following command:
+```bash
+$ mkdir -p "${HOME}/.config/gtk-4.0" && ln -sf "${THEME_DIR}/gtk-4.0/assets" "${HOME}/.config/gtk-4.0/assets" && ln -sf "${THEME_DIR}/gtk-4.0/gtk.css" "${HOME}/.config/gtk-4.0/gtk.css" && ln -sf "${THEME_DIR}/gtk-4.0/gtk-dark.css" "${HOME}/.config/gtk-4.0/gtk-dark.css"
+```
+Make sure to export `$THEME_DIR` to where the theme is before executing the command.
+
+###  Applying the theme on Flatpak (Skip this step if flatpak isn't installed)
+If you have Flatpak installed, make sure to execute the following commands to theme flatpak with the theme.
+
+In order for flatpak to access the theme, make sure to run the following command:
+```bash
+$ sudo flatpak override --filesystem=$HOME/.themes
+```
+
+Then, run the following command to apply the theme.
+```bash
+$ sudo flatpak override --env=GTK_THEME=$THEME_DIR
+```
+
+Again, Make sure to export `$THEME_DIR` to where the theme is before executing the command or the theme won't work.
+
+### Using the install script to install the theme
+
+To install the theme using the install script, run ``install.py`` using Python with
+```
+$ python install.py <latte, frappe, macchiato, mocha> <accent color>
+```
+If you have adwaita installed, make sure to include --link in order to add symlinks for it
+```
+$ python install.py <latte, frappe, macchiato, mocha> <accent color> --link
+```
+Run the command and the gtk theme should be installed!
 
-We have 4 AUR packages for all the 4 flavours of the theme:
+### Using the AUR to install the theme
 
-- [Latte](https://aur.archlinux.org/packages/catppuccin-gtk-theme-latte)
-- [Frappe](https://aur.archlinux.org/packages/catppuccin-gtk-theme-frappe)
-- [Macchiato](https://aur.archlinux.org/packages/catppuccin-gtk-theme-macchiato)
-- [Mocha](https://aur.archlinux.org/packages/catppuccin-gtk-theme-mocha)
+We have 4 AUR packages for all the 4 flavours (Latte, Frappe, Macchiato, Mocha)
 
-With your favourite AUR helper, install them:
+With your favourite AUR helper, you can install one of these flavors:
 
 ```bash
-yay -S catppuccin-gtk-theme-mocha catppuccin-gtk-theme-macchiato catppuccin-gtk-theme-frappe catppuccin-gtk-theme-latte
+$ yay -S catppuccin-gtk-theme-<flavor>
```

Comment:
```suggestion
yay -S catppuccin-gtk-theme-<flavor>
```

We should avoid needlessly prefixing commands with `$` as it makes commands more difficult to copy and paste.

Reply:
im removing it rn